app/views/content/service_privacy_notice_candidate.html.erb

```diff
@@ -110,13 +110,32 @@
       by the Department for Education. This is to support your application journey. For example, we’ll send you
       relevant information and support opportunities.</p>
 
+    <p class="govuk-body">We will also use the phone number you provide to contact you about whether Get Into Teaching’s
+    teacher training advisers can support you in your application.</p>
+
+    <h3 class="govuk-heading-s">If you’re successful</h3>
+
     <p class="govuk-body">If you’re successfully recruited to a teacher training course, your data will also be shared
-      between Apply and Register trainee teachers, which are both delivered by the Department for Education. This is
-      to allow you to be registered as a trainee teacher for the start of your teacher training course.</p>
+    between Apply and Register trainee teachers, which are both delivered by the Department for Education. This is
+    to allow you to be registered as a trainee teacher for the start of your teacher training course.</p>
 
     <p class="govuk-body">If you’re successfully recruited to a teacher training course, your data will also be shared between Apply and Teaching Vacancies, which are both delivered by the Department for Education.
-      This will allow us to contact you so we can provide support as you look for a role upon qualifying as a teacher.
-      Sign up to the service is voluntary.</p>
+    This will allow us to contact you so we can provide support as you look for a role upon qualifying as a teacher.
```

Comment:
```suggestion
    We’ll contact you to ask whether you’d like to sign up to Teaching Vacancies, so that we can support you to find a teaching job. There is no obligation to sign up. 
```